lib/screens_web/views/v2/audio/reconstructed_alert_fullscreen_view.ex

```diff
@@ -0,0 +1,206 @@
+defmodule ScreensWeb.V2.Audio.ReconstructedAlertFullscreenView do
+  use ScreensWeb, :view
+
+  alias Screens.Alerts.Alert
+  alias Screens.Util
+
+  def render("_widget.ssml", alert) do
+    ~E|<p><%= render_banner(alert) %><%= render_alert(alert) %></p>|
+  end
+
+  def render_banner(%{
+        unaffected_routes: _unaffected_routes
+      }),
+      do: nil
+
+  def render_banner(%{
+        region: _region,
+        routes: routes
+      }) do
+    if routes |> hd() |> Map.has_key?(:headsign) do
+      destinations = Enum.map(routes, fn route -> route.headsign end)
```

Comment:
So if the first map has a `:headsign` key, they all do?

Reply:
Yeah, that's right. Either they all get condensed to just simple RL / BL route pills, or they have direction included. [Miro](https://miro.com/app/board/uXjVM-ve3yE=/?moveToWidget=3458764557094546650&cot=14)